api/src/main/java/org/pmiops/workbench/disks/DiskAdminService.java

```diff
@@ -78,27 +75,23 @@ public DiskAdminService(
     this.fireCloudService = fireCloudService;
     this.initialCreditsService = initialCreditsService;
     this.leonardoApiClient = leonardoApiClient;
-    this.leonardoMapper = leonardoMapper;
     this.mailService = mailService;
     this.userDao = userDao;
     this.workspaceDao = workspaceDao;
   }
 
-  public void checkPersistentDisks() {
-    // Fetch disks as the service, which gets all disks for all workspaces.
-    final List<ListPersistentDiskResponse> disks = leonardoApiClient.listDisksAsService();
```

Comment:
Previously, this was launched from a Leonardo call so it operated on the Leonardo-entity `ListPersistentDiskResponse`.  Now it is called via the Task Queues endpoint, which uses the RWB-entity `TaskQueueDisk`.

Most of the changes here are related to that translation.